Modules/HR/Config/config.php

```diff
@@ -224,10 +224,16 @@
         'scanned_copy_pan_card' => 'scanned copy of Pan Card',
         'passbook_first_page_img' => 'Passbook First page IMG',
     ],
+
     'encrypted-applicant-details' => [
         'aadhar_card_number' => 'Aadhar Card Number',
         'pan_card_number' => 'Pan Card Number',
         'acc_number' => 'Account Number',
         'ifsc_code' => 'IFSC Code',
     ],
+
+    'hr-email' => [
+        'primary' => env('HR_FOLLOWUP_EMAIL_PRIMARY'),
```

Comment:
Keep it as default as well
```suggestion
        'primary' => env('HR_FOLLOWUP_EMAIL_PRIMARY', 'hr@coloredcow.com'),
```